server/src/main/java/io/deephaven/server/console/ConsoleServiceGrpcImpl.java

```diff
@@ -137,6 +137,7 @@ public void startConsole(StartConsoleRequest request, StreamObserver<StartConsol
     @Override
     public void subscribeToLogs(LogSubscriptionRequest request, StreamObserver<LogSubscriptionData> responseObserver) {
         GrpcUtil.rpcWrapper(log, responseObserver, () -> {
+            sessionService.getCurrentSession();
```

Comment:
Not a particular criticism of your change, but given that we often use this to assert authentication, maybe we should rename it?

Reply:
I think we might want to have an additional method with a more appropriate name, yes.